Nitaicandra/CENPROJECT
# Home view + User forms fix (state field) + service listing page

## Summary of Changes
- Created a home view based on roles, i.e customer vs business
- added context functionality so logged in user info is easily accessible by all components in the application
- users that are not logged-in are redirected to login page
- Updated backend to meet frontend's requirements

## How to test
Customer 
1. Log in as a customer. The browser should automatically redirect you to /home
2. The customer home page should show a search bar and any upcoming bookings
3. Try to search for a service and click on view details to see that service's page. The book and business buttons are not yet implemented (book is pending approval of PR 24)

Business
1.  Log in as a business. The browser should automatically redirect you to /home
2. The business home should show a button to create a new service and any upcoming bookings.
3. Clicking on the new service button should redirect you to new create service page.
4. The upcoming bookings also shows two buttons for additional functionality for businesses: delete and edit a booking. These are fully functional and should work as expected.
